lib/operations/db_collection_ops.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+const applyWriteConcern = require('../utils').applyWriteConcern;
+const Collection = require('../collection');
+const executeCommand = require('./db_execute_commands').executeCommand;
```

Comment:
tbh, this is why I wanted to keep the operations relegated to their own huge file first (`database_ops.js`). It's not immediately clear how much overlap there is between these files yet, and it becomes cumbersome to figure that out when we arbitrarily split it up. It's also not immediately obvious which file I need to look in for which operation.